### PR TITLE
update symfony console version

### PR DESCRIPTION
**Pull request by kocoten1992**
update symfony console version

### CODE REVIEW

**Comment by kocoten1992** [comment]
hi @povils , I've update dependencies, should we go with symfony 4.x version ?

**Comment by victorlap** [comment]
Why not use php 7.1.3? As this is the version `symfony/console` requires, see: https://github.com/symfony/console/blob/3179d1350843d2caa796973173dd0609ed4c7c67/composer.json#L19

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,7 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
-  - hhvm
+  - 7.2
```
Comment (peter279k) [inline .travis.yml]: I think the `php-7.1` is required because this version is still under official PHP support.

File: composer.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
     }
   ],
   "require": {
-    "php": ">=5.4",
-    "symfony/console": "~3.0"
+    "php": ">=7.2.0",
```
Comment (peter279k) [inline composer.json]: It should set the `>=7.1` version.

**Comment by devnix** [comment]
> This package requires the `php-7.1` version at least because this version is still under official PHP support.

At the time of your comment PHP 7.1 was already without active support (January 1, 2019) and will be out of life on January 1, 2020. A merge of this PR would be greatly appreciated!

**Comment by kocoten1992** [comment]
I'm sorry, lost track of this one, please open another pull request, thanks and apologize :disappointed:

**Pull request closed without merging**